stdlib/src/builtin/float_literal.mojo

```diff
@@ -435,7 +435,95 @@ struct FloatLiteral(
         """
         return rhs // self
 
-    # TODO - maybe __pow__?
+    @always_inline("nodebug")
+    fn __exp__[atol: FloatLiteral = 0.1e-20](self) -> FloatLiteral:
```

Comment:
Thanks for this work. A small nit is that `atol` is a function that acts on numbers, so I think we should spell out the whole word `absolute_tolerance`.

Reply:
yeah i agree